pkg/apis/core/types.go

```diff
@@ -5453,6 +5453,18 @@ type Preconditions struct {
 	UID *types.UID
 }
 
+// LogStreamType represents the desired log stream type.
+type LogStreamType string
+
+const (
+	// LogStreamTypeStdout is the stream type for stdout.
+	LogStreamTypeStdout LogStreamType = "stdout"
+	// LogStreamTypeStderr is the stream type for stderr.
+	LogStreamTypeStderr LogStreamType = "stderr"
+	// LogStreamTypeAll represents the combined stdout and stderr.
```

Comment:
We should probably mention that this is the default.

Reply:
The default value is mentioned in the comment of the `Stream` field of `PodLogOptions` now